docker-compose.yml

```diff
@@ -467,6 +467,26 @@ services:
     ports:
       - "15000:15000"
 
+  #
+  # The second version of "live replay" server
+  #
+  faf-aio-replayserver:
+    container_name: faf-aio-replayserver
+    build: git://github.com/FAForever/faf-aio-replayserver
+    image: faforever/faf-aio-replayserver:0.1
+    user: ${FAF_AIO_REPLAYSERVER_USER}
+    restart: unless-stopped
+    volumes:
+      - ./data/content:/content
+    networks:
+      - faf
+    depends_on:
+      - faf-init-volumes
+      - faf-db
```

Comment:
Depending on `faf-db` is outdated, you can removed it.
Otherwise, we might accidentally restart the DB on the server